### DISCUSSION TITLE
Apple QuickLook and Environement textures

### DISCUSSION THREAD

**Discussion by FrostKiwi** [google/model-viewer#4849] [Q&A]
### Description
I'm looking to clarify the following understanding:

Using Model Viewer in Apple Quicklook mode, you cannot change the the IBL Environment. As per Apple Documentation: https://developer.apple.com/documentation/arkit/arkit_in_ios/specifying_a_lighting_environment_in_ar_quick_look it is predefined by Apple to be [studio_lighting_armode_v002.exr](https://developer.apple.com/sample-code/ar/studio_lighting_armode_v002.exr)

Apple does support WebXR on Vision Pro, but not iPads and iPhones and probably never will, so you gotta use QuickLook there. There are no workarounds.

You can define Environment Textures in in model-viewer, but they will be ignored, as soon as you enter Apple QuickLook. 
Did I get the current state of technology right?

**Comment by elalish** [comment]
I believe that's all correct, and I hadn't realized they published their IBL - that's nice. @AdaRoseCannon in case Apple wants to add anything.
